pkg/instancetype/instancetype.go

```diff
@@ -312,6 +282,27 @@ func (m *methods) StoreControllerRevisions(vm *virtv1.VirtualMachine) error {
 	return nil
 }
 
+func storeRevision(revision *appsv1.ControllerRevision, clientset kubecli.KubevirtClient) (*appsv1.ControllerRevision, error) {
+	foundRevision, err := clientset.AppsV1().ControllerRevisions(revision.Namespace).Create(context.Background(), revision, metav1.CreateOptions{})
+	if err != nil {
+		if !errors.IsAlreadyExists(err) {
+			return nil, fmt.Errorf("failed to create ControllerRevision: %w", err)
+		}
+
+		// Grab the existing revision to check the data it contains
+		existingRevision, err := clientset.AppsV1().ControllerRevisions(revision.Namespace).Get(context.Background(), revision.Name, metav1.GetOptions{})
+		if err != nil {
+			return nil, fmt.Errorf("failed to get ControllerRevision: %w", err)
+		}
+		// If the data between the two differs return an error, otherwise continue and store the name below.
```

Comment:
Can you drop the `and store the name below` from this comment, it doesn't make sense now given it has moved.

Reply:
Again, please amend the comment given the changes in the series.

Reply:
Done